zproject/default_settings.py

```diff
@@ -207,6 +207,7 @@
 ENABLE_GRAVATAR = True
 INLINE_IMAGE_PREVIEW = True
 INLINE_URL_EMBED_PREVIEW = True
+REQUIRE_UNIQUE_NAMES = False
```

Comment:
I don't think we need a server-level setting for this; it seems like it should work fine as realm-level configuration.

(The similar settings for changing full names predate the organization-level setting, and would not have been added were it not for that)